test/test_robot_wrapper.py

```diff
@@ -407,3 +417,119 @@ def test_fetch_robot_wrapper(fixed_base):
                 "test_fetch_robot_wrapper__fixed_base=" + str(fixed_base),
                 open_vid=True,
             )
+
+
+@pytest.mark.skipif(
+    not osp.exists("data/robots/hab_franka"),
+    reason="Test requires Fetch robot URDF and assets.",
+)
+def test_franka_robot_wrapper():
+    """Test the franka robot."""
+    # set this to output test results as video for easy investigation
+    produce_debug_video = False
+    observations = []
+    cfg_settings = default_sim_settings.copy()
+    cfg_settings["scene"] = "NONE"
+    cfg_settings["enable_physics"] = True
+
+    # loading the physical scene
+    hab_cfg = make_cfg(cfg_settings)
+
+    with habitat_sim.Simulator(hab_cfg) as sim:
+        obj_template_mgr = sim.get_object_template_manager()
+        rigid_obj_mgr = sim.get_rigid_object_manager()
+
+        # setup the camera for debug video (looking at 0,0,0)
+        sim.agents[0].scene_node.translation = [0.0, -1.0, 2.0]
+
+        # add a ground plane
+        cube_handle = obj_template_mgr.get_template_handles("cubeSolid")[0]
+        cube_template_cpy = obj_template_mgr.get_template_by_handle(
+            cube_handle
+        )
+        cube_template_cpy.scale = np.array([5.0, 0.2, 5.0])
+        obj_template_mgr.register_template(cube_template_cpy)
+        ground_plane = rigid_obj_mgr.add_object_by_template_handle(cube_handle)
+        ground_plane.translation = [0.0, -0.2, 0.0]
+        ground_plane.motion_type = habitat_sim.physics.MotionType.STATIC
+
+        # compute a navmesh on the ground plane
+        navmesh_settings = habitat_sim.NavMeshSettings()
+        navmesh_settings.set_defaults()
+        sim.recompute_navmesh(sim.pathfinder, navmesh_settings, True)
+        sim.navmesh_visualization = True
+
+        # add the robot to the world via the wrapper
+        robot_path = "data/robots/hab_franka/robots/panda_arm.urdf"
```

Comment:
Similarly, check that this path is correct. I think it might be "data/robots/franka_panda/panda_arm.urdf"